src/ansi-c/expr2c.cpp

```diff
@@ -30,6 +30,12 @@ Author: Daniel Kroening, kroening@kroening.com
 #include "c_qualifiers.h"
 #include "expr2c_class.h"
 
+expr2c_configurationt expr2c_configurationt::default_configuration{true,
```

Comment:
Is this really what clang-format is doing? :/

Reply:
Yes I know :cry:

Reply:
A case for `// clang-format off`, `// clang-format on`?